dockstore-common/src/main/java/io/dockstore/common/Utilities.java

```diff
@@ -82,13 +83,16 @@ public static INIConfiguration parseConfig(String configFile) {
     }
 
     public static ImmutablePair<String, String> executeCommand(String command) {
-        return executeCommand(command, true, Optional.of(ByteStreams.nullOutputStream()), Optional.of(ByteStreams.nullOutputStream()), null);
+        return executeCommand(command, null);
```

Comment:
Hmmm, not sure. 
But I think this means the default for all commands is now 15 seconds rather than just nextflow parsing in the webservice. For example workflow launches via cromwell or cwltool

Please confirm